lessons/pydata/pandas_core/index.ipynb

```diff
@@ -1769,6 +1769,13 @@
     "**Úkol:** Co se stane, když totéž učiníš se `Series`?"
```

Comment:
<p>Udělali(y) by to manuálně (bez použití max / sort apod.)? Jinak mě nenapadá, jak to udělat s tím co se tady ukazuje. Kdyžtak bych to nějak upřesnil / naznačil.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
<p>Jj, .set_index() a .sort_index() je v hore v lekcii Indexovanie.</p><p> </p><p>Takže si stačí potom tento df zobraziť a ukáže sa ti posledných 5 riadkov (ak sa nám niekto nebude hrabať v Pandas options :-D ).</p><p> </p><p>Prípadne by som mohol nechať len tú časť na speed a to by si mohol spraviť zobrazením Series.</p><p> </p>